geopandas/io/file.py

```diff
@@ -34,15 +35,15 @@ def _is_url(url):
         return False
 
 
-def read_file(filename, bbox=None, mask=None, rows=None, **kwargs):
+def read_file(filepath_or_buffer, bbox=None, mask=None, rows=None, **kwargs):
```

Comment:
We certainly can't change the name of argument as it would break backward compatibility.

Reply:
Got it... I actually changed it to emulate pandas